src/test/kotlin/org/jmailen/gradle/kotlinter/support/RuleSetsTest.kt

```diff
@@ -13,29 +12,24 @@ class RuleSetsTest {
 
     @Test
     fun `resolveRuleSets loads from classpath providers`() {
-        val result = resolveRuleSets(defaultRuleSetProviders)
+        val standardOnly = resolveRuleProviders(defaultRuleSetProviders, includeExperimentalRules = false)
+        val withExperimentalRules = resolveRuleProviders(defaultRuleSetProviders, includeExperimentalRules = true)
 
-        assertEquals(listOf("standard"), result.map { it.id })
-    }
-
-    @Test
-    fun `resolveRuleSets loads from classpath providers including experimental rules`() {
```

Comment:
Given we now return a flat set of all rule providers, I couldn't figure out how to test the order, so I removed this test and modified the one above to ensure the `includeExperimentalRules` works in an expected way